### PR TITLE
Docs: review GRPC GSG

### PR DESCRIPTION
**Pull request by eloycoto**
Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

### CODE REVIEW

File: Documentation/gettingstarted/grpc.rst [hunk 1]
```diff
@@ -16,70 +16,81 @@ Step 1: Installing Cilium
 The next step is to install Cilium into your Kubernetes cluster.  Cilium installation
```
Comment (aanm) [inline Documentation/gettingstarted/grpc.rst]: @eloycoto remove this whole `Step 1` section and replace it with 
```
.. include:: cilium_install.rst
```

**Review by aanm — CHANGES_REQUESTED** [review]

File: Documentation/gettingstarted/grpc.rst [hunk 2]
```diff
@@ -10,76 +10,52 @@ minutes.
...
-    secret "cilium-etcd-secrets" created
-    daemonset "cilium" created
-
-Kubernetes is now deploying Cilium with its RBAC, ConfigMap and Daemon Set as a
-pod on minikube. This operation is performed in the background.
-
-Run the following command to check the progress of the deployment:
+.. include:: cilium_install.rst
 
+It is important for this demo that ``kube-dns`` is working correctly, to know the
```
Comment (ianvernon) [inline Documentation/gettingstarted/grpc.rst]: is working correctly. To know the status...

File: Documentation/gettingstarted/grpc.rst [hunk 3]
```diff
@@ -10,76 +10,52 @@ minutes.
...
-    $ kubectl get ds --namespace kube-system
-    NAME            DESIRED   CURRENT   READY     NODE-SELECTOR   AGE
-    cilium          1         1         0         <none>          2m
+    $ kubectl get deployment kube-dns -n kube-system
+    NAME       DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+    kube-dns   1         1         1            1           13h
 
-Wait until the cilium Deployment shows a ``CURRENT`` count of ``1`` like above (a ``READY`` value of ``0`` is OK
-for this tutorial).
+Where at least one pod should be Availabe.
```
Comment (ianvernon) [inline Documentation/gettingstarted/grpc.rst]: Available --> available.

File: Documentation/gettingstarted/grpc.rst [hunk 4]
```diff
@@ -10,76 +10,52 @@ minutes.
...
 
 Step 2: Deploy the Demo Application
 ===================================
 
 Now that we have Cilium deployed and ``kube-dns`` operating correctly we can
 deploy our demo gRPC application.  Since our first demo of Cilium + HTTP-aware security
-policies was Star Wars-themed we decided to do the same for gRPC.  While the
+policies was Star Wars-themed we decided to do the same for gRPC. While the
```
Comment (ianvernon) [inline Documentation/gettingstarted/grpc.rst]: ... Star Wars-themed, we decided...

File: Documentation/gettingstarted/grpc.rst [hunk 5]
```diff
@@ -158,17 +133,16 @@ parameters (in this case, the door-id):
...
-and help the rebels escape.   
-
-To see this, run: 
-
+to Cloud City identify and locate different doors. But recall that the door service also
+exposes several other methods, including ``SetAccessCode``. If access to the door manager
+service is protected only using traditional IP and port-based firewalling, the TCP port of
+the service (50051 in this example) will be wide open to allow legitimate calls like
+``GetName`` and ``GetLocation``, which also leave more sensitive calls like ``SetAccessCode`` exposed as
+well. Its this mismatch between the course granularity of traditional firewalls and
```
Comment (ianvernon) [inline Documentation/gettingstarted/grpc.rst]: Its --> It is

**Review by ianvernon — CHANGES_REQUESTED** [review]
Minor nitpicks / grammatical corrections.

**Comment by eloycoto** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Pull request merged**